autospec/build.py

```diff
@@ -196,20 +194,19 @@ def get_uniqueext(dirn, dist, name):
     return "{}-{}".format(name, seq)
 
 
-def set_mock():
-    global mock_cmd
-    # get group list of current user
-    user_grps = [grp.getgrgid(g).gr_name for g in os.getgroups()]
-    if os.path.exists('/usr/bin/mock'):
-        if 'mock' not in user_grps:
-            mock_cmd = 'sudo /usr/bin/mock'
+def get_mock_cmd():
```

Comment:
Could you please document briefly (e.g: why we do) the difference between just calling `/usr/bin/mock` vs `sudo /usr/bin/mock`.

something like
```
# if using consolehelper (fedora, and other distros), user must be in the mock
# group to gain privileges. On Clearlinux we use 'sudo' to avoid the use of
# consolehelper, so user in mock group is unnecessary
```
would do fine, so for future reference we understand this better just by reading the comment at the top of the function.